pontoon/sync/management/commands/import_pootle.py

```diff
@@ -0,0 +1,153 @@
+from bulk_update.helper import bulk_update
+from collections import Counter
+from datetime import datetime
+from django.contrib.auth.models import Permission, User
+from django.core.management.base import BaseCommand, CommandError
+from django.utils import timezone
+from pontoon.base.models import Translation, Project, Locale
+
+import requests
+
+
+class Command(BaseCommand):
+    help = 'Import translation authors and dates from Verbatim.'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--project',
+            action='store',
+            dest='project',
+            help='Project slug to import Verbatim data to'
+        )
+
+        parser.add_argument(
+            '--locale',
+            action='store',
+            dest='locale',
+            help='Locale code to import Verbatim data to'
+        )
+
+    def handle(self, *args, **options):
```

Comment:
You should wrap this whole thing in `transaction.atomic` so that if it fails the changes are rolled back.